gallery/psgallery/psgallery_deploy_to_azure_automation_requireLicenseAcceptance.md

```diff
@@ -0,0 +1,22 @@
+---
+ms.date:  2017-06-12
+contributor:  JKeithB
```

Comment:
>JKeithB [](start = 14, length = 7)

This should be your name, right?

Reply:
Yes. Good Catch. Fixed :)

---
In reply to: [138457062](https://github.com/PowerShell/PowerShell-Docs/pull/1620#discussion_r138457062) [](ancestors = 138457062)